src/FffPolygonGenerator.cpp

```diff
@@ -345,12 +357,24 @@ void FffPolygonGenerator::processBasicWallsSkinInfill(SliceDataStorage& storage,
     
     
     // walls
-    for (unsigned int layer_number = 0; layer_number < mesh.layers.size(); layer_number++)
+    int processed_layer_count = 0;
+#pragma omp parallel for default(none) shared(mesh_layer_count, mesh, inset_skin_progress_estimate, processed_layer_count) schedule(dynamic)
+    for(unsigned int layer_number = 0; layer_number < mesh.layers.size(); layer_number++)
```

Comment:
Code style: Space after for keyword. Fixed that for you in 2d9a3c83efe6b6d59c6314ba12243fd6260382be.